lib/cog/command/pipeline/initializer.ex

```diff
@@ -17,19 +17,19 @@ defmodule Cog.Command.Pipeline.Initializer do
   alias Cog.Repository.ChatHandles
   alias Cog.Passwords
 
-  defstruct mq_conn: nil, history: %{}, history_token: ""
+  defstruct mq_conn: nil, history: %{}, previous_command: ""
```

Comment:
This should be `previous_command_prefix` or `previous_command_token`, etc.  It might get a little confusing if people think this should literally hold the last command that was executed.